modules/test/playwright/tests/frontend-data-set-web/clayTable.spec.ts

```diff
@@ -18,13 +18,12 @@ export const test = mergeTests(
 	apiHelpersTest,
 	featureFlagsTest({
 		'LPS-178052': true,
-		'LPS-193005': true,
 	}),
 	isolatedSiteTest,
 	loginTest()
 );
 
-test('Sort columns', {tag: '@LPS-193005'}, async ({apiHelpers, page, site}) => {
```

Comment:
Hey @larissacribeiro, for historical purposes, you could keep these tags here. They serve as a reference for the tickets that motivated those tests.

Reply:
Hi @igor-franca, sure! I will keep that in mind for future reference because I though that meant that this test would run under that feature flag. In this case, this change of mine won't be used since I'm not responsible for removing the feature flag, it was just for the purpose of running the tests. But thank you!!